mutator/statement/remove.go

```diff
@@ -37,6 +39,11 @@ func MutatorRemoveStatement(pkg *types.Package, info *types.Info, node ast.Node)
 		l = n.Body
 	}
 
+	containsPanic := strings.Contains(mutesting.GetNodeASTString(node), "panic")
+	if containsPanic && len(l) == 1 {
+		return nil
+	}
```

Comment:
So this says that "Do not remove this statement, if anything inside of it contains a panic"?

Reply:
Seems a bit concerning if its arbitrarily deep.

if its only of length 1-2 deep, sounds great!

Reply:
Just moved the length check before the AST panic search so it should now only run on AST leaves!

I also added thorough comments to this filter & the error check one explaining what they do in greater detail